packages/arb-token-bridge-ui/src/hooks/useTheme.ts

```diff
@@ -1,34 +1,30 @@
 import { useEffect } from 'react'
-import { useArbQueryParams } from './useArbQueryParams'
+import useLocalStorage from '@rehooks/local-storage'
+
+export const themeLocalStorageKey = 'arbitrum:bridge:preferences:theme'
 
 export const classicThemeKey = 'arbitrum-classic-theme'
 
 export const THEME_CONFIG = [
   {
     id: 'space',
     label: 'Space',
-    queryParam: '',
     description:
       'A dark, space-themed UI with a sleek and futuristic aesthetic, featuring Arbinaut on a backdrop of shining stars and moon.'
   },
   {
     id: classicThemeKey,
     label: 'Arbitrum Classic',
-    queryParam: classicThemeKey,
     description:
       'Arbitrum before it was cool: A reminiscent of the pre-nitro era, with simple, solid buttons, a minimal purple layout and chunky fonts.'
   }
 ]
 
 export const useTheme = () => {
-  const [{ theme }, setQueryParams] = useArbQueryParams()
-
-  const setTheme = (_theme: string) => {
-    setQueryParams({ theme: _theme })
-  }
+  const [theme, setTheme] = useLocalStorage<string>(themeLocalStorageKey)
 
   useEffect(() => {
-    document.body.className = theme
+    document.body.className = theme || ''
```

Comment:
slightly strange as it's not obvious that this is added because `theme` might be `null`
maybe `if (!theme) return` before this line?

Reply:
Yeah, that makes sense. Good point.